### DISCUSSION TITLE
关于萨卡兹自动肉鸽的一个问题

### DISCUSSION THREAD

**Discussion by rockazhou** [MaaAssistantArknights/MaaAssistantArknights#10122] [General]
首先，非常感谢MAA的自动肉鸽功能，对于没时间玩肉鸽的粥友而言是福音。
在长时间使用萨卡兹自动肉鸽（一直挂机到目前65级奖励拿满），发现其只在初始难度通关过一次一结局，其余难度下均没有通关记录。
经观察发现，自动肉鸽在到达5层boss房门口时，队伍的人数与练度均不高，有时甚至只有ew一个精二。
推测是一局肉鸽中，每次遇到战斗关就会尝试刷新掉，导致招募次数减少，最终造成了这样的局面。
这对于自动刷等级、刷投资、刷技能树没有很大的影响，但对于自动化和电子斗蛐蛐爱好者而言不友好（之前观察水月肉鸽能逐渐解锁更高难度，很开心），且干员招募数量、精二数量、是否通关也会影响最终分数评定，侧面影响刷分效率。
建议：能否取消刷新战斗关的机制（或只刷新高难关），并相应修改思绪过载时的处理方法。
虽然是老博士，但抛弃大脑久矣，对于干员、肉鸽、MAA等不甚了解。如果工程实现有难度，或有其他考量，无法修改，请忽视此反馈。打扰了。
最后，非常感谢MAA项目组。MAA是在繁忙的工作生活中维系我与粥的重要纽带，真的帮到我很多。

**Comment by Saratoga-Official** [comment]
maa只有选了蓝图分队才会刷新，不让maa刷新可以用职业分队

**Comment by Daydreamer114** [comment]
没人写作战策略
